### DISCUSSION TITLE
[924970] Back 4 Blood

### DISCUSSION THREAD

**Discussion by SZCDSB** [pjy612/SteamManifestCache#44] [Share]
appid:924970
keys:
783767a483d935235bc2a46262a58b4acc3524bdf6d52116f090fcccc0a89024
[924970_config.zip](https://github.com/pjy612/SteamManifestCache/files/13686570/924970_config.zip)
code:AE442CF7CCD49880
email:szcdsb@gmail.com

**Comment by pjy612** [comment]
查收
